packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -14794,28 +14794,33 @@ testWidgets('Floating cursor ending with selection', (WidgetTester tester) async
     });
 
     testWidgets(
-      'Error thrown when spell check enabled but no default spell check service available',
+      'Spell check disabled when spell check enabled but no default spell check service available',
```

Comment:
"Spell check disabled when spell check enabled" is contradictory, maybe instead say: "Spell check disabled when spellCheckConfiguration enabled"